utils/potential_utf/src/uchar.rs

```diff
@@ -108,8 +105,16 @@ impl PotentialCodePoint {
     /// ```
     #[inline]
     pub unsafe fn to_char_unchecked(self) -> char {
-        let [u0, u1, u2] = self.0;
-        char::from_u32_unchecked(u32::from_le_bytes([u0, u1, u2, 0]))
+        char::from_u32_unchecked(u32::from(self))
+    }
+
+    /// For converting to the ULE type in a const context
+    ///
+    /// Can be removed once const traits are a thing
+    #[inline]
+    #[cfg(feature = "zerovec")]
+    pub const fn const_to_unaligned(self) -> zerovec::ule::RawBytesULE<3> {
```

Comment:
Nit: We've started doing the pattern of shadowing the trait functions when we need them to be const so long as the signatures match.